src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -244,22 +257,25 @@ private void pausePopup(final int buttonNo) {
 
 	private void playerDies() {
 		State.getTrack().getPlayer().die();
-		sleep(2 * InternalTicker.NANOS_PER_TICK / InternalTicker.E6);
+		sleep(20 * InternalTicker.NANOS_PER_TICK / InternalTicker.E6);
```

Comment:
20 is a magic number. Please check your tests with SAT.

<review,configuration